packages/next/components/postDetail/biasedVotingItem.js

```diff
@@ -0,0 +1,52 @@
+import styled from "styled-components";
+import Tooltip from "../tooltip";
+import ValueDisplay from "../valueDisplay";
+
+const Wrapper = styled.div`
+  position: relative;
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
+`;
+
+const LabelWrapper = styled.div`
+  color: #506176;
+  position: relative;
+  display: flex;
+  width: 60%;
+
+  // for tooltip with size=fit
+  > *:first-child {
+    max-width: 100%;
+  }
+
+  .label {
```

Comment:
class defination here isn't combined with any other styled-component, may just difine a `const Label = style.div` and apply these css within it